test/test_lattice.py

```diff
@@ -19,7 +19,7 @@ def simple_element(identity=1):
     sp_pv = 'setpoint_pv'
     device1 = pml.device.Device(mock.MagicMock(), sp_pv, rb_pv)
     device2 = pml.device.Device(mock.MagicMock(), sp_pv, rb_pv)
-    element.add_to_family('family')
+    element.add_to_family('BPM')
```

Comment:
You didn't really need to change the family name from `family` to `BPM`.  It doesn't have to be a real family name, any string will do.